### ISSUE TITLE
Allow re-banning a player that is already banned

### ISSUE THREAD

**Issue by nokoa** [BanManagement/BanManager#219]
Please allow banning a player that is already banned, this feature is needed in order to bump time severity of a punishment. 

For Example;
A low level staff is only permitted to ban for short terms. If longer time is needed for the punishment, the low level staff will request a punishment bump from a higher staff member and that player will re-ban the player for the appropriate time with a different reason.

Basically, when re-banning it will unban the player and ban them again.

This feature is already present for Mutes.

Regards,
Nokoa.

**Comment by confuser** [comment]
I'm not sure why you think this is a feature already present for mutes, as it really shouldn't be, and if it is, I'd consider it as a bug.

New permissions will be added, bm.command.override

**Comment by Agazed** [comment]
Please add this!
